src/core/components/layout/components/container/container.tsx

```diff
@@ -1,27 +1,49 @@
 import { HTMLAttributes, ReactNode } from 'react';
 import { SerializedStyles } from '@emotion/react';
-import { container, containerBorder } from './styles';
+import {
+	container,
+	containerSideBorders,
+	containerTopBorder,
+	containerBackground,
+} from './styles';
 import { Props } from '@guardian/src-helpers';
+import { border } from '@guardian/src-foundations/palette';
 
 interface Container extends HTMLAttributes<HTMLDivElement>, Props {
-	border?: boolean;
```

Comment:
I would suggest leaving the `border` prop in place for backward compatibility for now. We can remove it from the documentation to ensure people don't use it in the future.